feat: no found message added

snoli2001 opened the pull request.

snoli2001 committed:
feat: no found message added

cpantu commented:
Seba, el PR lo veo bien, lo único que me parece medio jugado es el echo que asumimos que todos los autocomplete va a mostrar este texto,  cuando antes no mostrábamos nada. 

Creo que lo mas prudente seria agregar un booleano en extra que determine si debe o no mostrar el texto, inclusive también agregaría en el extra la posibilidad de cambiar el texto, herald esta en ingles pero tenemos plataformas que estan en español y debemos tener forma de cambiar el texto. 

¿Qué te parece?

snoli2001 commented:
@cpantu ok, agrego las modificaciones 💪

snoli2001 committed:
show not found message logic updated

Por defecto no se mostrará el mensaje y ahora se podrá mandar un mensaje personalizado. Si se desea probar, en el boton de "Prueba de Modal API" estará disponible.

snoli2001 commented:
@cpantu agregué las modificaciones

aeserein-tots commented:
@snoli2001 Está bien. Lo que le faltaría para aprobarlo es que se pueda setear por provider. No tengo problema en hacerlo o en dejar que lo hagas, pero hay un error pendiente en la forma en que están puestos los providers en este repositorio. También hay que hacer el mismo pr y correcciones en la versión 16. Lo dejo abierto hasta entonces. Después de Furniture voy a tener tiempo
